exo-setup/src/cli.ls

```diff
@@ -14,7 +14,7 @@ module.exports = ->
 
   app-config = yaml.safe-load fs.read-file-sync('application.yml', 'utf8')
   console.log "Setting up #{green app-config.name} #{cyan app-config.version}\n"
-  logger = new Logger flatten [Object.keys(app-config.services[type]) for type of app-config.services]
+  logger = new Logger flatten [Object.keys(app-config.services[type]) unless app-config.services[type] is null for type of app-config.services]
```

Comment:
We don't care if the services are `null`, `undefined`, or an empty array. Hence, this change can be simplified from `unless app-config.services[type] is null` to `if app-config.services[type]`